python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -2454,4 +2454,4 @@ def get_time_partition_key(
             partitions_def.time_window_dimension.name
         ]
     else:
-        check.failed(f"Cannot get time partition from non-time partitions def {partitions_def}")
+        check.failed(f"Cannot get time partition from non-time partitions def {partitions_def}")
```

Comment:
@tacastillo I backed out the changes I made and replaced this entire file with the version from master. I'm honestly not sure what's going on here 😭